src/tests/Common/Coreclr.TestWrapper/MobileAppHandler.cs

```diff
@@ -191,5 +176,36 @@ private static void CreateRetryFile(string fileName, int exitCode, string appNam
                 writer.WriteLine($"appName: {appName}; exitCode: {exitCode}"); 
             }
         }
+
+        private static int[] GetKnownExitCodes()
```

Comment:
Why not a static field?

Reply:
Then I would need to create an instance of this class, which seems to be unnecessary for what I want to do here.

Reply:
Functionality-wise, a static method just operates on input parameters and do not have to get or set any internal instance fields. Design-wise, I don't see a need to preserve that piece of data in an instance of class, as I don't even create an instance of this class at the callsite.

Reply:
Static fields get initialized with the static ctor the first time you run any static method on that type. I think functionality wise there is no difference, but each time we at first need to call `GetKnownExitCodes()` which then initializes the same array over and over again on the heap and GCs it after, right? So why not just use a field we have ready on the static type?

But it's not a big deal and doesn't matter in the grand scheme of things at all, I was just a bit surprised we need a whole method for this.

Reply:
Oh, I see. You meant to replace this function with a static field... That makes sense. Let me change that!